src/file-upload/file-upload.styles.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import { Alert } from "../alert";
+import { Text, TextStyleHelper } from "../text";
+import { MediaQuery } from "../media";
+import { Button } from "../button";
+import { Color } from "../color";
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const TitleContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 2rem;
```

Comment:
does margin bottom need to remain when both title and description are not specified?

Reply:
Nope. Will conditionally render them